samples/yarn-pnp/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@vitest/vscode-sample-yarn-pnp",
```

Comment:
Corepack is confused locally, so probably we need to add `packageManager` explicitly here:

```diff
diff --git a/samples/yarn-pnp/package.json b/samples/yarn-pnp/package.json
index a1ee270..68c9864 100644
--- a/samples/yarn-pnp/package.json
+++ b/samples/yarn-pnp/package.json
@@ -3,6 +3,7 @@
   "type": "module",
   "private": true,
   "version": "1.0.0",
+  "packageManager": "yarn@4.1.1",
   "scripts": {
     "test": "vitest"
   },
```

Reply:
Agreed. Will do.